### PR TITLE
Update to support packaged releases

### PR DESCRIPTION
**Pull request by kris-work**
## What was the problem/requirement? (What/Why)
Some small updates were required to support a more streamlined package/release process.

## What was the solution? (How)
- Made updates to plugin metadata.
- Moved AWSAmbit.hda to top-level of plugin folder.

## What artifacts are related to this change?
Internal AWS documentation for "Ambit Deployment Runbook".

## What is the impact of this change?
Creating and distributing release builds will be a bit easier.

## Are you adding any new dependencies to the system?
No

## How were these changes tested?
I validated that the changes work with the AWS team's internal documentation for creating release builds.

----
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by eherozhao — APPROVED** [review]
Looks good to me.

File: Ambit/Config/FilterPlugin.ini [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[FilterPlugin]
+; This section lists additional files which will be packaged along with your plugin. Paths should be listed relative to the root plugin directory, and
+; may include "...", "*", and "?" wildcards to match directories, files, and individual characters respectively.
+;
+; Examples:
+;    /README.txt
+;    /Extras/...
+;    /Binaries/ThirdParty/*.dll
+AWSAmbit.hda
```
Comment (brhook-aws) [inline Ambit/Config/FilterPlugin.ini:9]: Shouldn't this be `/HoudiniDigitalAssets/AWSAmbit.hda` or other variant with wildcard?

**Review by haozhao-aws — APPROVED** [review]
Looks good to me.

File: Ambit/Config/FilterPlugin.ini (hunk 1 above)
Comment (kris-work) [reply Ambit/Config/FilterPlugin.ini:9]: Nope. Included in this PR is a move of that .hda up a level to simplify packaging. There is no "HoudiniDigitalAssets" folder anymore. Also, I made the conscious decision not to use a wildcard for now in favor of making inclusion of this and any future HDAs in the packaged build explicit. We can always change that decision later if the need arises.

**Pull request merged**